S/SuiteSparse@5/build_tarballs.jl

```diff
@@ -17,7 +17,7 @@ cd $WORKSPACE/srcdir/SuiteSparse/
 # Apply Jameson's shlib patch
 atomic_patch -p1 ${WORKSPACE}/srcdir/patches/SuiteSparse-shlib.patch
 # Disable OpenMP as it will probably interfere with blas threads and Julia threads
-FLAGS=(INSTALL="${prefix}" INSTALL_LIB="${libdir}" INSTALL_INCLUDE="${prefix}/include" CFOPENMP=)
+FLAGS=(INSTALL="${prefix}" INSTALL_LIB="${libdir}" INSTALL_INCLUDE="${prefix}/include" MY_METIS_LIB="-lmetis" CFOPENMP=)
```

Comment:
Is `MY_METIS_LIB` used anywhere?

Reply:
Oh, it's in the Makefiles.

Reply:
SuiteSparse_config should pick it up.

https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/79f25b523ae0fd81abe1ffb0efd9005f2e6eef33/SuiteSparse_config/SuiteSparse_config.mk#L491